src/PlaybackManager.vala

```diff
@@ -455,4 +457,30 @@ public class Music.PlaybackManager : Object {
         var play_pause_action = (SimpleAction) GLib.Application.get_default ().lookup_action (Application.ACTION_PLAY_PAUSE);
         play_pause_action.set_enabled (current_audio != null);
     }
+
+        public void save_queue () {
```

Comment:
Seems like there's some funny indentation here? Also since this isn't called from outside of this class, it should be private